app/Http/Controllers/Administration/UserController.php

```diff
@@ -139,4 +139,48 @@ public function getEmail(): array
 			'email' => $user->email,
 		];
 	}
+
+	/**
+	 * Returns the currently authenticated user.
+	 *
+	 * @return User
+	 */
+	public function getAuthenticatedUser(): User
+	{
+		/** @var User */
+		return Auth::user();
+	}
```

Comment:
Do we still need this method? (More a reminder for myself to check it later.)

Reply:
I don't mind whether we keep it or not. It's your call.

But if we keep it, we should have one or two tests for it, i.e. one test which checks for the unauthenticated user that the JSON response is `null`, another test which checks for an authenticated user that the expected user object is returned. Without tests this method is just dead code and will become an "unbeloved child" in future updates. I agree that this method is so trivial, that it hardly will ever contain a bug, but the devil is a squirrel. Moreover, me secret goal is to push test coverage over 85%. So we should test what we can test and here this method even implements an external interface.